assets/js/admin/layouts.js

```diff
@@ -1,59 +1,8 @@
-/* global jQuery, primer_layouts_transport */
+/* global jQuery */
 window.wp = window.wp || {};
 
 ( function( $ ) {
 
-	var api = wp.customize;
```

Comment:
@EvanHerman This was removed because we no longer have the layouts option in the customizer.